src/web/src/app/c/[[...id]]/Components/ChatUi/ListenModeRenderer.scss

```
@@ -130,8 +130,10 @@
 }
 
 .listen-reveal-wrapper:not(.mobile):not(.listen-reveal-wrapper--classroom) {
+  --listen-playback-timeline-height: 34px;
   --listen-slide-player-bottom-offset: calc(
-    var(--chat-footer-height, 40px) + var(--listen-player-footer-gap, 12px)
+    var(--chat-footer-height, 40px) + var(--listen-player-footer-gap, 12px) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)